system-parachains/constants/src/lib.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub mod kusama;
+pub mod polkadot;
+
+pub use constants::*;
+
+/// re-export types from upstream
+pub use parachains_common::BlockNumber;
+
+/// Common constants of parachains.
+mod constants {
```

Comment:
I think we should not name things by kind. Having a module named "enums" or "structs" would be equally weird - especially since its all re-exported anyway.

It could be ordered by topic (time, currency, fee...), however i think its not worth it since the file is so small.

Reply:
Happy to change, I have no attachment to how it was in `polkadot-sdk`